lib/models/device.ts

```diff
@@ -3392,18 +3392,24 @@ const getDeviceModel = function (
 			): Promise<void> {
 				value = String(value);
 
-				const deviceFilter = isId(uuidOrId)
-					? uuidOrId
-					: {
-							$any: {
-								$alias: 'd',
-								$expr: {
-									d: {
-										uuid: uuidOrId,
-									},
+				let deviceFilter;
```

Comment:
This atm is typed as `any`, so let's make it:
```suggestion
				let deviceFilter: ResourceObjFilter<Device>[keyof ResourceObjFilter<Device>];
```

Reply:
nice catch